djangoevents/apps.py

```diff
@@ -75,6 +75,9 @@ def new_init(self, *args, **kwargs):
         old_init(self, *args, **kwargs)
 
         if adds_schema_version_to_event_data():
-            self.__dict__['schema_version'] = get_event_version(self.__class__)
+            dct = self.__dict__
+            key = 'schema_version'
+            if key not in dct:
+                dct[key] = get_event_version(self.__class__)
```

Comment:
Why not simply:

```python
 if 'schema_version' not in self.__dict__:
    ...
```

Reply:
Could make a typo in the string this way. 😛  This might be an iOS habit. In our Swift app all keys are static struct properties (namespaced constants, effectively).